lib/managers/remote_config_manager.dart

```diff
@@ -1,11 +1,17 @@
 import 'package:api/dependency_injection.dart';
+
+import 'package:api/models/app/managers/connection_status_manager.dart';
 import 'package:api/models/app/managers/remote_config.dart';
 
 final RemoteConfigManager _remoteConfigManager =
     DependecyInjection.instance.get<RemoteConfigManager>();
 
 Future<void> activateRemoteConfig() async {
-  await _remoteConfigManager.initializeRemoteConfig();
-  _remoteConfigManager.setDefaultParams();
-  _remoteConfigManager.launchLoadingStrategy();
-}
+  final ConnectionStatusManager connectionStatusManager =
+      DependecyInjection.instance.get<ConnectionStatusManager>();
```

Comment:
Get the connectionStatusManager above the function. Like the "_remoteConfigManager". 
And make it private